docs/migrations.md

```diff
@@ -151,6 +151,11 @@ Example with `ts-node`:
 ts-node ./node_modules/typeorm/cli.js migration:run
 ```
 
+Example with `ts-node` using `yarn v2` with `pnp`:
```

Comment:
This isn't Yarn v2 nor PnP specific, it should be done like this no matter what version of Yarn and/or if you're using `node_modules`. If you're using workspaces `node_modules/typeorm/cli.js` might not exist as it could have been hoisted or a future version of `typeorm` change the bin path.

Reply:
Updated; please resolve if you're good.

Reply:
Looks good 👍; I don't have access to resolve